apps/admin_api/test/admin_api/v1/controllers/wallet_controller_test.exs

```diff
@@ -175,6 +177,7 @@ defmodule AdminAPI.V1.WalletControllerTest do
                            "metadata" => %{},
                            "encrypted_metadata" => %{},
                            "enabled" => true,
+                           "avatar" => expected_avatar,
```

Comment:
I would avoid having to reference another variable as it's harder to see the whole response structure. This is where copy & paste works better.

Also applies to the rest of the tests.